idaes/core/util/tests/test_pid_initialization.py

```diff
@@ -36,7 +36,11 @@
 )
 from idaes.core.util.model_statistics import degrees_of_freedom
 from idaes.models.unit_models import CSTR, Mixer, MomentumMixingType
-from idaes.models.control import PIDBlock, PIDForm
+from idaes.models.control.controller import (
```

Comment:
Looks like you could import directly from ```idaes.models.control```, not that it matters, just a little shorter.